sdk/eventhub/event-processor-host/changelog.md

```diff
@@ -1,89 +1,108 @@
+### 2.1.1
+
+- Updates minimum version of `@azure/event-hubs` to version 2.1.4.
+  This update brings in improvements when attempting to reconnect after a transient failure.
```

Comment:
should we point to the event hubs changelog and call out that there have been improvements in the past 3 releases etc

Reply:
Sure, I'll point to the changelog. I thought I had called out that there were improvements though :)

Reply:
Updated

Reply:
Yes, you did :)
Something to denote that the improvement is not just in 2.1.4 but in the releases leading up to it is useful. 
That way, when pointed to the changelog, one would scroll down and read as well :)